src/kagglehub/handle.py

```diff
@@ -9,6 +9,8 @@
 NUM_VERSIONED_MODEL_PARTS = 5  # e.g.: <owner>/<model>/<framework>/<variation>/<version>
 NUM_UNVERSIONED_MODEL_PARTS = 4  # e.g.: <owner>/<model>/<framework>/<variation>
 
+NUM_DATASET_PARTS = 2  # e.g.: <owner>/<datasetname>
```

Comment:
There are versioned and unversioned datasets urls as well.  This probably needs to follow the same pattern as the `MODEL_PARTS` above.

Also, tiny comment suggestion for consistency/clarity:

```suggestion
NUM_DATASET_PARTS = 2  # e.g.: <owner>/<dataset>
```